sound/soc/sof/sof-client-ipc-msg-injector.c

```diff
@@ -18,16 +18,19 @@
 #include <sound/sof/ipc4/header.h>
 
 #include "sof-client.h"
+#include "ipc3-priv.h"
```

Comment:
subject: "ASoC: SOF: ipc-msg-injector: Add support for injecting message for the kernel"